src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,71 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
-    public void getStatistic(String fromFileName, String toFileName) {
+    private static final int INDEX_OF_SUPPLY_LENGTH = 7;
+    private static final int INDEX_OF_BUY_LENGTH = 4;
 
+    public void getStatistic(String fromFileName, String toFileName) {
```

Comment:
This method is overcomplicated. Let's divide it into three private methods. 
- method which will read String data from file
- method which will transform data info statistic
- and a method which will write data info file

Reply:
question about methods in the slack